templates/python/.github/workflows/publish.yaml

```diff
@@ -0,0 +1,37 @@
+name: Publish
+# Template adapted from https://github.com/bakdata/ci-templates/tree/main/.github/workflows#python-poetry-release
+
+on:
+  workflow_dispatch:
+    inputs:
+      release-type:
+        description: "Scope of the release; see https://python-poetry.org/docs/cli/#version"
+        type: choice
+        required: true
+        default: patch
+        options:
+          - patch
+          - minor
+          - major
+    
+jobs:
+  release:
+    name: release
+    uses: bakdata/ci-templates/.github/workflows/python-poetry-release.yaml@v1.13.5
+    with:
+      release-type: ${{ github.event.inputs.release-type }}
+      publish-to-test: true
+      python-version: "3.10"
+      # poetry-version: 1.1.11
+      # working-directory: "./my-awesome-python-project"
+      github-email: ${{ secrets.GH_EMAIL }}
+      github-username: ${{ secrets.GH_USERNAME }}
+      github-token: ${{ secrets.GH_TOKEN }}
+      pypi-token: ${{ secrets.TEST_PYPI_TOKEN }}
+
+  print-version:
+    name: print-version
+    runs-on: ubuntu-latest
```

Comment:
Same here, please pin to the most recent version

Reply:
done